hack/update-deps.sh

```diff
@@ -28,7 +28,7 @@ VERSION="master"
 # The list of dependencies that we track at HEAD and periodically
 # float forward in this repository.
 FLOATING_DEPS=(
-  "knative.dev/pkg@${VERSION}"
+  "knative.dev/pkg@release-0.17"
```

Comment:
I believe that line 50 will need a similar change, or the mix of variable vs. constant should be juggled.

Reply:
Done.